src/pdfium.rs

```diff
@@ -303,14 +303,14 @@ impl Pdfium {
     /// * Embed the bytes of the target document directly into the compiled WASM module
     /// using the `include_bytes!()` macro.
     #[cfg(not(target_arch = "wasm32"))]
-    pub fn load_pdf_from_reader<R: Read + Seek + 'static>(
-        &self,
+    pub fn load_pdf_from_reader<'a, R: Read + Seek + 'a>(
```

Comment:
the most important change! 

the rest is just trickling stuff through from here into FpdfFileAccessExt